docs/2notebook/2_Constraints.ipynb

```diff
@@ -79,27 +79,57 @@
   {
    "cell_type": "code",
    "execution_count": 1,
-   "metadata": {
```

Comment:
I think the change in this notebook is by mistake? We shouldn't delete its original ouputs